lib/Models/Cesium3DTilesCatalogItem.ts

```diff
@@ -43,6 +48,11 @@ class ObservableCesium3DTileset extends Cesium3DTileset {
   }
 }
 
+interface ObservableCesium3DTileset {
+  modelMatrix: Matrix4;
```

Comment:
Why are we storing a separate copy of the tileset's modelMatrix? is it for reactivity? Or some other reason?

Reply:
Also I'm not super keen on defining a class and an interface with the same name in the same file

Reply:
Noted! This is just to make the type checker happy. `modelMatrix` should ideally be defined in the cesium typings.

Reply:
Fixed by moving the extra typings to the terriajs-cesium typings file.